src/main/java/uk/gov/di/ipv/cri/passport/lambda/PassportHandler.java

```diff
@@ -7,30 +7,32 @@
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.nimbusds.jose.JOSEException;
+import com.nimbusds.jose.JWSObject;
 import com.nimbusds.oauth2.sdk.AuthorizationCode;
 import com.nimbusds.oauth2.sdk.ParseException;
-import com.nimbusds.oauth2.sdk.id.Identifier;
 import com.nimbusds.openid.connect.sdk.AuthenticationRequest;
 import org.apache.http.HttpStatus;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import uk.gov.di.ipv.cri.passport.domain.DcsPayload;
+import uk.gov.di.ipv.cri.passport.domain.DcsResponse;
+import uk.gov.di.ipv.cri.passport.domain.DcsSignedEncryptedResponse;
+import uk.gov.di.ipv.cri.passport.domain.PassportFormRequest;
 import uk.gov.di.ipv.cri.passport.error.ErrorResponse;
+import uk.gov.di.ipv.cri.passport.exceptions.HttpResponseExceptionWithErrorBody;
 import uk.gov.di.ipv.cri.passport.helpers.ApiGatewayResponseGenerator;
-import uk.gov.di.ipv.cri.passport.persistence.item.DcsResponseItem;
+import uk.gov.di.ipv.cri.passport.persistence.item.PassportCheckDao;
 import uk.gov.di.ipv.cri.passport.service.AuthorizationCodeService;
+import uk.gov.di.ipv.cri.passport.service.ConfigurationService;
+import uk.gov.di.ipv.cri.passport.service.DcsCryptographyService;
 import uk.gov.di.ipv.cri.passport.service.PassportService;
-import uk.gov.di.ipv.cri.passport.validation.ValidationResult;
 
 import java.io.IOException;
 import java.security.KeyStoreException;
 import java.security.NoSuchAlgorithmException;
 import java.security.cert.CertificateException;
 import java.security.spec.InvalidKeySpecException;
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
+import java.util.*;
```

Comment:
Can we swap this back to individual imports? I'm guessing this is just the annoying intellij setting doing this

Reply:
Yes, my silly IntelliJ settings. Fixed for now.

Reply:
Yeah this might be someone's intellij settings, go to `preferences -> Editor -> Code Style -> Java -> General -> Class count to use import with '*' and Names count to use static import with '*'` and set them to `999`.